scripts/twister

```diff
@@ -193,6 +193,7 @@ if not ZEPHYR_BASE:
     print(f'ZEPHYR_BASE unset, using "{ZEPHYR_BASE}"')
 
 sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/pylib/twister/"))
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/pylib/helpers"))
```

Comment:
I think `build_helpers` would be clearer than `helpers`. Especially `helpers/domains.py` feels a bit vague.